src/hotspot/share/opto/superword.cpp

```diff
@@ -3615,3 +3030,279 @@ bool SuperWord::same_origin_idx(Node* a, Node* b) const {
 bool SuperWord::same_generation(Node* a, Node* b) const {
   return a != nullptr && b != nullptr && _clone_map.same_gen(a->_idx, b->_idx);
 }
+
+void SuperWordVTransformBuilder::build_vtransform() {
```

Comment:
Note: used to be `PacksetGraph::build`. I also decided to refactor its content into smaller functions.